dashboard/app/views/layouts/application.html.haml

```diff
@@ -51,7 +51,7 @@
 
     #gdpr-dialog
 
-    %script{src: webpack_asset_path('js/code-studio.js'), data: {gdpr: gdpr_data.to_json, usertype: user_type.to_json}}
+    %script{src: webpack_asset_path('js/code-studio.js'), data: {gdpr: gdpr_data.to_json, usertype: user_type.to_json, currentUserId: current_user.id}}
```

Comment:
unfortunately this is setting us up for problems because some pages (cached HoC script levels) will be sent a currentUserId but then will cache it in a way that other users would incorrectly get the first user's id: https://github.com/code-dot-org/code-dot-org/blob/120a31ae7ba7ab67316c543c79c7bcf068ab30e6/cookbooks/cdo-varnish/libraries/http_cache.rb#L191-L195

Because we do not currently need to solve for providing user id on cached HOC script levels, I am going to suggest that you pass the user id down only to the specific pages which need it.